android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -550,25 +653,27 @@ public void receiveCommand(WebView root, int commandId, @Nullable ReadableArray
           RNCWebView reactWebView = (RNCWebView) root;
           JSONObject eventInitDict = new JSONObject();
           eventInitDict.put("data", args.getString(0));
+          // Make sure this is the same as iOS!
```

Comment:
I'm not sure this is related to this PR + this is deprecated and will be removed soon so I don't really see a reason to change it

Reply:
This is going to be removed? Are you just expecting this to be implemented by individuals themselves?

Reply:
I can remove if you want, but I am relying on this for my app. It was changes on iOS but not Android so now they're inconsistent which is awkward.

Reply:
For some context #438

Reply:
Well that work can always be done on top of this one...